### PR TITLE
New Distribution. HA and non-root

### PR DESCRIPTION
**Pull request by eldada**
Add support for the new Distribution 1.2.0.
Two major changes:
1. Support HA (Distribution service)
2. Non-root Docker containers

### CODE REVIEW

**Comment by jainishshah17** [comment]
@eldada @omerkay We will need to remove our current logic for creating configMap for the token. When we use this logic it adds newline char at end of file. Which changes checksum and gives an error in authenticating requests made by Distributor to Distribution.

**Comment by jainishshah17** [comment]
I propose asking user to use `kubectl cp` command to move the token from Distribution to Distributor.

**Comment by eldada** [comment]
@jainishshah17 - the issue with the newline in the token was fixed in the distribution code. @omerkay and I confirmed the new chart works as expected with token this way.

**Comment by jainishshah17** [comment]
screen shot 2018-09-04 at 3 01 02 pm

**Comment by jainishshah17** [comment]
I am still able to reproduce the token issue. Refer to above screenshot to see new line char at end of file.

File: stable/distribution/values.yaml [hunk 1]
```diff
@@ -74,9 +74,9 @@ mongodb:
   mongodbExtraFlags:
   - "--wiredTigerCacheSizeGB=1"
   mongodbRootPassword:
+  mongodbDatabase: bintray
```
Comment (liorhson) [inline stable/distribution/values.yaml]: mongo database name should be 'distribution'

Comment (eldada) [reply stable/distribution/values.yaml]: @liorhson see https://github.com/jfrog/charts/pull/27

**Pull request merged**